README.rst

```diff
@@ -84,8 +119,8 @@ function ``hello_world`` in ``myapp.api``. Optionally you can include ``x-swagge
           x-swagger-router-controller: myapp.api
           operationId: hello_world
 
-
-To customize this behavior, Connexion can use alternative ``Resolvers``, for example ``RestyResolver``. The ``RestyResolver`` will compose an ``operationId`` based on the path and HTTP method of the endpoints in your specification:
+**Automatic Routing**
```

Comment:
You need `:` and a empty line here, or maybe this should a subsection ([with `~~~~~~~~~` below the title](http://docutils.sourceforge.net/docs/user/rst/quickstart.html#sections))